packages/contracts-bedrock/test/libraries/Bytes.t.sol

```diff
@@ -68,6 +68,9 @@ contract Bytes_slice_Test is Test {
     /// @notice Tests that the `slice` function correctly updates the free memory pointer depending
     ///         on the length of the slice.
     function testFuzz_slice_memorySafety_succeeds(bytes memory _input, uint256 _start, uint256 _length) public {
+        _start = bound(_start, 0, _input.length - 1);
+        _length = bound(_length, 0, _input.length - _start);
+
         // The start should never be more than the length of the input bytes array - 1
         vm.assume(_start < _input.length);
         // The length should never be more than the length of the input bytes array - the starting
```

Comment:
We should be able to remove both `vm.assume`s here as they are now no-ops

Reply:
I left them out of being worried that the `bound` call was not exactly correct